website/docs/d/is_security_group_rules.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_security_group_rules"
+description: |-
+  Get information about SecurityGroupRuleCollection
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm_is_security_group_rules
+
+Provides a read-only data source for SecurityGroupRuleCollection. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_is_security_group_rules" "example" {
+	security_group = ibm_is_security_group.example.id
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+- `security_group` - (Required, Forces new resource, String) The security group identifier.
```

Comment:
Mostly we don't specify Forces new resource.. please check the existing datasources..

Reply:
done.